.github/setup_ci.sh

```diff
@@ -142,6 +142,10 @@ for TARGET in $TARGETS; do
         INSTALL_BORINGSSL=1
         PACKAGES="${PACKAGES} cmake ninja-build"
        ;;
+    aws-lc)
+        INSTALL_AWSLC=1
+        PACKAGES="${PACKAGES} cmake ninja-build golang"
```

Comment:
golang is not needed unless it needs to run AWS-LC's tests. Add `-DBUILD_TESTING=OFF` to the cmake setup?